src/js/components/Button.js

```diff
@@ -51,7 +51,7 @@ var Button = React.createClass({
 
     return (
       <button id={this.props.id} type={type} className={classes.join(' ')}
-        onClick={this.props.onClick}>
+        onClick={this.props.onClick} disabled={! this.props.onClick}>
```

Comment:
this should be set, based on props, not based on the existance of a handler function. 
Consider the login form for example, I'd like to be able to set a button to `disabled` while the form is invalid. 
The above would force me to dynamically add a handler function to achieve this.